### PR TITLE
Don't leak our Python

### PR DESCRIPTION
**Pull request by hynek**
Is this what you meant @henryiii?

Or should I remove the venv wholesale?

### CODE REVIEW

**Comment by henryiii** [comment]
Yes, that's what I meant. :+1:

File: action.yml [hunk 1]
```diff
@@ -20,11 +20,13 @@ runs:
   using: composite
   steps:
     - uses: actions/setup-python@v4
+      id: python
```
Comment (henryiii) [inline action.yml:23]: Might it be better to be more baip specific here?

Comment (hynek) [reply action.yml:23]: Ah you mean rename it to prevent clashes?

Comment (henryiii) [reply action.yml:23]: Yes, I'm assuming this `id` is available after the action runs from the surrounding workflow. So "python-baip" or something like that would be better.

**Review by henryiii — APPROVED** [review]
LGTM!

**Comment by hynek** [comment]
Thanks!

**Pull request merged**